config/routes.rb

```diff
@@ -2,10 +2,16 @@
 
 Rails.application.routes.draw do
   # For details on the DSL available within this file, see https://guides.rubyonrails.org/routing.html
-  match ':controller(/:action(/:id))', :via => [:get, :post]
   namespace 'api' do
     namespace 'v1' do
-      resources :products
+      resources :screenshots
+      get 'screenshots/new', to: 'screenshots#new'
+      get 'screenshots/show', to: 'screenshots#show'
```

Comment:
Since you already have `resources :screenshots` you dint need line 8 and 9 imo.

Reply:
Thanks for the review. I will make the respective changes.